src/components/projects/ProjectTile.js

```diff
@@ -22,14 +22,10 @@ import Separator from '../common/Separator'
 import PopupMessage from './PopupMessage'
 import theme from '../../theme'
 
-const mapStateToProps = (state, ownProps) => {
-    const completeness = Number.parseFloat(ownProps.project.completeness)
-    return {
-        tileWidth: state.main.device.width - 52,
-        containsNativeWorkflows: ownProps.project.workflows.length > 0,
-        outOfData: !Number.isNaN(completeness) && completeness >= 1
-    }
-};
+const mapStateToProps = (state, ownProps) => ({
+    tileWidth: state.main.device.width - 52,
```

Comment:
What does 52 represent here? Might be good to have as a constant so it is more apparent.